composeApp/src/commonTest/kotlin/sp/bvantur/tasky/core/domain/ValidateNameUseCaseImplTest.kt

```diff
@@ -0,0 +1,48 @@
+package sp.bvantur.tasky.core.domain
+
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class ValidateNameUseCaseImplTest {
+
+    @Test
+    fun `GIVEN name is empty WHEN validating THEN validation fails`() {
+        val name = ""
+        val useCase = ValidateNameUseCaseImpl()
+
+        assertFalse(useCase(name))
+    }
+
+    @Test
+    fun `GIVEN name has only 3 characters WHEN validating THEN validation fails`() {
+        val name = "Tes"
+        val useCase = ValidateNameUseCaseImpl()
+
+        assertFalse(useCase(name))
+    }
+
+    @Test
+    fun `GIVEN name has only 51 characters WHEN validating THEN validation fails`() {
+        val name = "123456789012345678901234567890123456789012345678901"
+        val useCase = ValidateNameUseCaseImpl()
+
+        assertFalse(useCase(name))
+    }
```

Comment:
Such test cases that are coupled to the max length of a string I like to make dependent on the max length constant you have:
```
val name = buildString {
    repeat(MAX_LENGTH) {
        append("1")
    }
    append("1") // Exceed the max length
}
```
That way, the test won't break when requirements change

Reply:
Nice one! 🤩